helpers/deploy/deploymentUtils.ts

```diff
@@ -1,9 +1,10 @@
 import { ethers, getNamedAccounts } from "hardhat";
 
-import bscMainnetGovernanceDeployments from "../../deployments/bscmainnet.json";
 import bscTestnetGovernanceDeployments from "../../deployments/bsctestnet.json";
 import { SUPPORTED_NETWORKS } from "./constants";
 
+const ADDRESS_ONE = "0x0000000000000000000000000000000000000001";
```

Comment:
nit: This could be isolated to line 50 where there is a comment to remove it, since it shouldn't be used anywhere else

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/57/commits/130bac81b04c0b3996bc7c16d2e55c39e9b5c941